frontend/src/app/projects/create-project/create-project.component.html

```diff
@@ -38,19 +38,28 @@
               </mat-form-field>
             </fieldset>
             <fieldset>
-              <mat-form-field appearance="fill">
-                <mat-label>Visibility</mat-label>
-                <mat-select matInput formControlName="visibility" required>
-                  <mat-option value="internal">
-                    Internal - all logged in users can see the project and have
-                    read access on its models.
-                  </mat-option>
-                  <mat-option value="private">
-                    Private - users need to be explicitly granted access to see
-                    the project.
-                  </mat-option>
-                </mat-select>
-              </mat-form-field>
+              <fieldset>
+                <div class="flex flex-col">
+                  <span>Project visibility:</span>
```

Comment:
I would make this a header (without semicolon. Can change this in the internal-project branch.

Reply:
As the PR is already merged, would you go ahead and implement it in #842?